weblate_web/templates/snippets/service.html

```diff
@@ -1,107 +1,167 @@
+                            {% trans "You can use use support services on our webite. Or, for more comfort, use an activation token to connect your Weblate installation to the support portal. More in " %}
+                            <a href="https://docs.weblate.org/en/latest/admin/support.html" class="link">{% trans "Documentation." %}</a>
                         </p>
                 {% endif %}
                     </div>
                     <div class="clear"></div>
                 </div>
-                {% if service.note %}
-                <div class="form-line">
-                    <div class="line-left">{% trans "Note" %}</div>
-                    <div class="line-right">
-                        <p>{{ service.note }}</p>
-                    </div>
-                    <div class="clear"></div>
-                </div>
-                {% endif %}
                 {% with subscriptions=service.subscription_set.all %}
                 {% for subscription in subscriptions %}
-                    <div class="form-line">
+                    <div class="form-line style6">
                         <div class="line-left">{% trans "Subscription" %}</div>
                         <div class="line-right">
-                            <strong>{{ subscription.get_package_display }}</strong>
+                            <p>{{ subscription.get_package_display }} <em>—</em> {% if subscription.expires|days_diff_from_today < 365 %}
+                                <span>monthly</span>
+                                {% else %}
+                                <span>yearly</span>
+                                {% endif %}</p>
+                            <t>{{ subscription.created|date:"d M Y" }}<em>—</em><strong>{{ subscription.expires|date:"d M Y" }}</strong>
+                                <span>
+                                    {% if subscription.expires|days_diff_from_today < 30 %}
+                                    {% if subscription.expires|days_diff_from_today < 2 and subscription.expires|days_diff_from_today >= 0 %}
+                                    <t class="pl-1 orange">{{ subscription.expires|days_diff_from_today}} day remaining</t>
+                                    {% elif subscription.expires|days_diff_from_today < 0  %}
+                                    <t class="pl-1 orange">Expired</t>
+                                    {% else %}
+                                    <t class="pl-1 orange">{{ subscription.expires|days_diff_from_today}} days remaining</t>
+                                    {% endif %}
+                                    {% else %}
+                                    <t class="pl-1 grey">{{ subscription.expires|days_diff_from_today}} days remaining</t>
+                                    {% endif %}
+                                </span>
+                            </t>
+                            <br>
+                            <v style="font-size: 16px; color: #BFC3C7;">Note:
```

Comment:
Please avoid inline CSS.

Reply:
I must have left it there by mistake, i have gotten rid of that now.